py/htm/examples/rest/hotgym.py

```diff
@@ -0,0 +1,219 @@
+import csv
+import datetime
+import os
+import numpy as np
+import math
+
+from rest_api import Link, Region, NetworkConfig, Network, get_classifer_predict
+
+_EXAMPLE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+_INPUT_FILE_PATH = os.path.join(_EXAMPLE_DIR, "gymdata.csv")
+
+default_parameters = {
+    # there are 2 (3) encoders: "value" (RDSE) & "time" (DateTime weekend, timeOfDay)
+    'enc': {
+        "value": {
+            'resolution': 0.88,
```

Comment:
I haven't checked, but these params should match those of "pythonic hotgym.py" example.

Reply:
> I haven't checked, but these params should match those of "pythonic hotgym.py" example.

yes i copy from then py/htm/examples/hotgym.py